en-US/firefox/switch.lang

```diff
@@ -0,0 +1,75 @@
+## NOTE: url for demo server is: https://bedrock-demo-agibson.us-west.moz.works/firefox/switch/
+## URL: https://bedrock-demo-agibson.us-west.moz.works/%LOCALE%/firefox/switch/
+
+
+# HTML page title
+;Switch from Chrome to Firefox in just a few minutes
+Switch from Chrome to Firefox in just a few minutes
+
+
+# HTML page description
+;Switching to Firefox is fast, easy and risk-free, because Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.
+Switching to Firefox is fast, easy and risk-free, because Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.
+
+
+;Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.
+Switching to Firefox is fast, easy and risk-free. Firefox imports your bookmarks, autofills, passwords and preferences from Chrome.
+
+
+;Select what to take from Chrome.
+Select what to take from Chrome.
+
+
+;Let Firefox do the rest.
+Let Firefox do the rest.
+
+
+;Enjoy the web faster, all set up for you.
+Enjoy the web faster, all set up for you.
+
+
+# This is a link.
+;Download and switch
+Download and switch
+
+
+;Note:
+Note:
+
+
+;You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.
+You can use Firefox and still have Chrome. Chrome won’t change on your machine one bit.
+
+
+;Share with your friends how to switch to Firefox
+Share with your friends how to switch to Firefox
+
+
+;Share to Facebook
+Share to Facebook
+
+
+# The string below is for a referral email subject line.
+;Switch to Firefox
+Switch to Firefox
+
+
+;Hey, Firefox makes switching from Chrome really fast. I like it a lot, and you should try it. Check it out and let me know what you think:
```

Comment:
We have a technical problem on this one, and I need to talk with alex to figure it out.